tools/gmhazard_scripts/db_creation/im_csv_to_imdb.py

```diff
@@ -481,9 +484,12 @@ def run(
     # Wait for the worker processes to finish
     print("Collecting and writing station data")
     ray.wait(worker_run_ids, num_returns=len(worker_run_ids))
+    print(f"wtf")
 
     close_id = writer.close_db.remote()
+    print(f"wtf2")
     ray.wait([close_id], num_returns=1)
+    print(f"wtf3")
```

Comment:
Maybe these should go haha

Reply:
Haha yes that might be a sensible idea... good catch